docker-compose.test.ct.yaml

```diff
@@ -0,0 +1,93 @@
+version: '3.7'
+
+networks:
+  sls:
+
+services:
+  ct-tests-smoke:
+    build:
+      context: test/ct/
+      dockerfile: Dockerfile
+    depends_on:
+      - cray-sls
+    entrypoint: entrypoint.sh smoke -f smoke.json -u http://cray-sls:8376
+    networks:
+      - sls
+#  s3:
```

Comment:
SLS uses S3, the smoke tests did not.  I did find a image replacement for minio in artifactory, so left it in.  If we ever create functional ct tests, it will likely be needed.